### PR TITLE
Add Cypress Operator single install mode test

### PR DESCRIPTION
**Pull request by rhamilto**
Replace disabled Protractor-based test with new Cypress one.

Resolves https://issues.redhat.com/browse/CONSOLE-2482

### CODE REVIEW

File: frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-single-namespace.spec.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  afterEach(() => {
+    checkErrors();
+  });
+
+  after(() => {
+    cy.deleteProject(testName);
+    cy.logout();
+  });
+
+  const operatorPkgName = '3scale-community-operator';
```
Comment (spadgett) [inline frontend/packages/operator-lifecycle-manager/integration-tests-cypress/tests/operator-install-single-namespace.spec.ts:21]: We might consider installing a custom catalog source like we were before to lock to a specific operator version, but I think this is fine for now. We'll see if it becomes an issue.

**Review by spadgett — COMMENTED** [review]
We'll need to update the test-prow-e2e.sh script to run this test when `olmFull` is passed as an argument. That will enable the Cypress OLM tests in the OLM merge queue.

https://github.com/openshift/console/blob/master/test-prow-e2e.sh#L44-L48

**Comment by spadgett** [comment]
We should remove the legacy Protractor test as well.

**Comment by rhamilto** [comment]
/hold

there be bugs

**Review by spadgett — COMMENTED** [review]
edit: Nevermind, the tests are in the right place.

/cc @dtaylor113

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**